tools/csv-merger/csv-merger.go

```diff
@@ -600,8 +335,6 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 	// Used by UI to validate user uploaded metadata
 	// Metadata is used to connect to an external cluster
 	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rbd"], "cephClusters": ["monitoring-endpoint"]}`
-	// Injecting the RHCS exporter script present in Rook CSV
-	ocsCSV.Annotations["external.features.ocs.openshift.io/export-script"] = rookCSV.GetAnnotations()["externalClusterScript"]
```

Comment:
thanks !
can you please create corresponding console BZ with the details (assuming PR will be merged in 4.16) ?? cc @Nikhil-Ladha @iamniting

Reply:
Since, this is for 4.16 I think having a JIRA story should be enough to take this in rather than creating a separate bz for it. Does that work for you @SanjalKatiyar ?

Reply:
it is a small change from UI, so we could have taken it up after feature freeze as well (hence the BZ) + this way, QE can test this flow once too !
or is this change already part of some 4.16 epic ??

Reply:
Yeah, it's a part of 4.16 epic that's the reason I suggested to have a JIRA story instead of a bz.
Ref: https://issues.redhat.com/browse/RHSTOR-5267